crates/core/src/rlp/decode.rs

```diff
@@ -375,6 +375,71 @@ pub fn decode_rlp_item(data: &[u8]) -> Result<(bool, &[u8], &[u8]), RLPDecodeErr
     }
 }
 
+/// Splits an RLP item in two:
+/// - The first item including its prefix
+/// - The remaining bytes after the item
+/// It returns a 3-element tuple with the following elements:
+/// - A boolean indicating if the item is a list or not.
+/// - The payload of the item, including its prefix.
+/// - The remaining bytes after the item.
+pub fn get_item_with_prefix(data: &[u8]) -> Result<(bool, &[u8], &[u8]), RLPDecodeError> {
```

Comment:
We aren't using the boolean so we should remove it.

Reply:
Done!